recipe/meta.yaml

```
@@ -23,12 +23,13 @@ requirements:
   run:
     - alembic
     - cliff
-    - cmaes
+    - cmaes >=0.5.1
     - colorlog
     - joblib
     - numpy
+    - packaging
```

A recent change in our `setup.py` https://github.com/optuna/optuna/pull/1599. How about incorporating this as well?
```suggestion
    - packaging >=20.0
```